### DISCUSSION TITLE
Deploy on IIS

### DISCUSSION THREAD

**Discussion by ceszgdav** [beyondcode/laravel-websockets#843] [Q&A]
I want to know how can i deploy the laravel websockets on a IIS Server, I have all my files up in it but in the docs and google I cant find something as guide to know how the deployment it is made it, the unique way I can call to the sockets is running the command on the cmd "php artisan websockets:serve"

**Comment by hpreethub** [comment]
HI

One way to do it using NSSM . create a service. run it and it will keep running in the background. If you made any changes in the application donot forget to restart the service
